tests/conftest.py

```diff
@@ -41,15 +42,15 @@ def _get_os_id():
 
     with open(OS_RELEASE_FILE) as f_in:
         for line in f_in:
-            match = re.search(r'^ID=[\'"]?(\w+)?[\'"]?$', line)
+            match = re.search(r'^ID=[\'"]?([\w_-]+)?[\'"]?$', line)
             if match:
                 os_id = match.group(1)
                 break
 
     return os_id
 
 
-_os_id = _get_os_id()
+_os_id = _get_os_id() or ()
```

Comment:
`_get_os_id()` returns the value `ID=` as string. `()` is tuple, right?
In a case, `_os_id` becomes string, in an another case `_os_id` becomes tuple, looks not great.
Is it better to change `()` to `''` (empty string)? How do you think?

```
>>> type(())
<class 'tuple'>
>>> type('')
<class 'str'>
```

Reply:
To be honest I do not remember why I added that line. I think it was to work around some issues with CentOS, but I've rebased those commits away already.

I've pushed a fixup commit that reverts it to the original state. I'm going to rebase it away, unless it breaks the testsuite, so please don't merge this yet 😉

Reply:
Sure. Perhaps, the issues with CentOS might be fixed by the latest commit of the master branch now. ( https://github.com/junaruga/rpm-py-installer/commit/8c8aad0669ba84255aadc13db425bec4ad241ce3 ).

When I was cherry-picking your commits, I faced CentOS6 case skipping `tests/test_install_fedora.py`. ( https://github.com/junaruga/rpm-py-installer/pull/211 )

Reply:
> Sure. Perhaps, the issues with CentOS might be fixed by the latest commit of the master branch now. ( [8c8aad0](https://github.com/junaruga/rpm-py-installer/commit/8c8aad0669ba84255aadc13db425bec4ad241ce3) ).

No it wasn't, my check in `_is_suse` was flawed and would raise an exception on CentOS6 (because `_os_id()` returns `None` there). I've tried a different version now.

Reply:
I've rebased the fixup and Travis should be hopefully green now.

Reply:
Sure, and Travis is green now :)